src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -130,24 +134,68 @@ interface SearchBarDropdownProps {
   isLoading: boolean
 }
 
-export const SearchBarDropdown = ({
-  toggleOpen,
-  tokens,
-  collections,
+const useSearchTrendingTokens = () => {
```

Comment:
originally i refactored this differently so the `SearchBarSuggestion` and this hook being split out are just remnants of that.. but not terrible